vuepress/docs/next/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -6,72 +6,76 @@ sidebarDepth: 2
 
 ## Overview
 
-This tutorial describes how to use the Entando Component Generator (ECG) to create microservices and micro frontends for deployment to the [Entando Component Repository](../../../docs/compose/local-hub-overview.md) and Entando Applications. The ECG is powered by [JHipster](https://www.jhipster.tech/) and leverages the Entando Blueprint. 
+This tutorial describes how to use the Entando Component Generator (ECG) to create microservices and micro frontends for deployment to the [Local Hub](../../../docs/compose/local-hub-overview.md) of an Entando Application or a shared [Entando Hub](../../../docs/getting-started/landing-page.md#entando.hub). The ECG is powered by [JHipster](https://www.jhipster.tech/) and leverages the Entando Blueprint. 
 
-![Entando Component Generator](./img/component-gen-flow.png)
+The [general development process](../../../docs/curate/bundle-details.md#bundle-development-process) is:
 
-The general flow of component generation is:
-
-1. Run the Entando Blueprint to create your components (Spring Boot microservice and optionally React micro frontends)
-2. Customize and enhance your generated code
-3. Build an Entando Bundle from your components
-4. Deploy a [custom resource](../../../docs/consume/custom-resources.md) for your bundle into Kubernetes
-5. Install your Entando Bundle into your Entando Application(s)
+1. Set up a new bundle project
+2. Run the Entando Blueprint to create your components (Spring Boot microservice and optionally React micro frontends)
+3. Configure the bundle project to wire together the components.
```

Comment:
we have been leaving bullets or numbered lists outlining a process unpunctuated on other pages. i think either way is fine, but suggested unpunctuated in this case because #3 is the only line item with a period, so changing it to align with the rest would be less effort than changing all the others to match